Bugsnag/Client/BugsnagClient.m

```diff
@@ -883,7 +796,7 @@ - (void)notifyOutOfMemoryEvent {
             // If the termination breadcrumb is set, the app entered a normal
             // termination flow but expired before the watchdog sentinel could
             // be updated. In this case, no report should be sent.
-            if ([name isEqualToString:kAppWillTerminate]) {
+            if ([name isEqualToString:@"App Will Terminate"]) {
```

Comment:
Maybe BSGNotificationBreadcrumbs.h should extern constants for this and the instance below?

Reply:
Done (in a slightly different way) 👍